### PR TITLE
US69765 - Convert all integration tests to use headers

### PR DESCRIPTION
**Pull request by FrankGasparovic**

### CODE REVIEW

File: commons/src/main/java/com/ge/predix/acs/utils/HttpServletRequestUtil.java [hunk 1]
```diff
@@ -1,50 +0,0 @@
-/*******************************************************************************
```
Comment (matiasa123) [inline commons/src/main/java/com/ge/predix/acs/utils/HttpServletRequestUtil.java:1]: Why did you delete these files?

Comment (matiasa123) [reply commons/src/main/java/com/ge/predix/acs/utils/HttpServletRequestUtil.java:1]: After speaking with frank the logic encapsulated in HttpServletRequestUtil was moved to tokenlib

**Review by matiasa123 — APPROVED** [review]

**Comment by FrankGasparovic** [comment]
Passed https://predix1.jenkins.build.ge.com/job/Predix-Security/ACS/acs-build-n-test-pullrequest/730//badge/icon
